src/plugins/hiddenColumns/test/plugins/manualColumnResize.e2e.js

```diff
@@ -0,0 +1,113 @@
+describe('HiddenColumns', () => {
+  const id = 'testContainer';
+
+  beforeEach(function() {
+    this.$container = $(`<div id="${id}"></div>`).appendTo('body');
+  });
+
+  afterEach(function() {
+    if (this.$container) {
+      destroy();
+      this.$container.remove();
+    }
+  });
+
+  describe('manualColumnResize', () => {
+    it('should resize a proper column when the table contains hidden column using mouse events', () => {
+      handsontable({
+        data: Handsontable.helper.createSpreadsheetData(2, 5),
+        colHeaders: true,
+        hiddenColumns: {
+          columns: [1],
+          indicators: true
+        },
+        manualColumnResize: true,
+      });
+
+      expect(colWidth(spec().$container, 1)).toBe(65); // 50 + 15 (indicator)
+
+      // Resize renderable column index `1` (within visual index term the index at 1 is hidden).
+      resizeColumn(1, 100);
```

Comment:
Could you change also the argument description in the `resizeColumn` function from the `common.js` file?

Reply:
Fixed within https://github.com/handsontable/handsontable/pull/6940/commits/dd3a20de608de1a3322c34d078f979494c1c794d.